Resources/doc/commands/generate_bundle.rst

```diff
@@ -19,14 +19,14 @@ structure:
 
 .. code-block:: bash
 
-    $ php app/console generate:bundle
+    $ php bin/console generate:bundle
```

Comment:
The comment above tells you to use bin instead of app for Symfony 3, so this is correct.